app/frontend/containers/EmailTargetView/EmailTargetView.js

```diff
@@ -0,0 +1,211 @@
+// @flow
+import React, { Component } from 'react';
+import _ from 'lodash';
+import $ from 'jquery';
+import { connect } from 'react-redux';
+import Select from '../../components/SweetSelect/SweetSelect';
+import './EmailTargetView.scss';
+import Input from '../../components/SweetInput/SweetInput';
+import Button from '../../components/Button/Button';
+import SelectCountry from '../../components/SelectCountry/SelectCountry';
+import { FormattedMessage } from 'react-intl';
+
+import {
+  changeCountry,
+  changeBody,
+  changeSubject,
+  changeSubmitting,
+  changePensionFunds,
+  changeEmail,
+  changeName,
+  changeFund,
+} from '../../state/email_target/actions';
+
+import type { Dispatch } from 'redux';
+
+class EmailTargetView extends Component {
+  componentDidMount() {
+    this.getPensionFunds(this.props.country);
+  }
+
+  getPensionFunds(country: string) {
```

Comment:
This returns undefined so it'd be difficult to test later on. Returning the promise and moving `handleSuccess` to its own method allows us to test both. In the `componentDidMount` fn, you can do 

```js
this.getPensionFunds(props.country)
  .then(mapPensionFundsData)
  .then(this.props.changePensionFunds);
```